architecture/modules/functions/main.tf

```diff
@@ -13,7 +13,7 @@ resource "google_cloudfunctions_function" "function" {
 }
 
 resource "google_cloudfunctions_function" "sign_files" {
-  name        = "sign_files"
+  name        = "sign-files"
```

Comment:
I was guilty of this with my last PR where I pinned dependency versions in an unrelated feature, but in the future, we should try to make our PRs as atomic as possible i.e. this should have been added to the naming changes PR imo.

Reply:
okay yes that is true. Do you think I should have opened a new PR for this?

Reply:
Yeah I think we should make separate PRs in the future :)

Reply:
Will do!